wire/msgtx.go

```diff
@@ -24,14 +24,26 @@ const (
 	// MaxPrevOutIndex is the maximum index the index field of a previous
 	// outpoint can be.
 	MaxPrevOutIndex uint32 = 0xffffffff
-)
 
-const (
-	// defaultTxInOutAlloc is the default size used for the backing array
-	// for transaction inputs and outputs.  The array will dynamically grow
-	// as needed, but this figure is intended to provide enough space for
-	// the number of inputs and outputs in a typical transaction without
-	// needing to grow the backing array multiple times.
+	// SequenceLockTimeDisabled is a flag that if set on a transaction
+	// input's sequence number, the sequence number will not be interpreted
+	// as a relative locktime.
+	SequenceLockTimeDisabled uint32 = 1 << 31
+
+	// SequenceLockTimeSeconds is a flag that if set on a transaction
+	// input's sequence number, the relative locktime has units of 512
+	// seconds.
+	SequenceLockTimeSeconds uint32 = 1 << 22
```

Comment:
This name is a bit ambiguous in my opinion.  It seems like it would refer to a quantity of seconds as compared to being a flag that indicates the sequence encodes a time-based relative locktime instead of a block-based one.  How about `SequenceLockTimeIsSeconds` or `SequenceLockTimeIsTimeBased`?